### ISSUE TITLE
Support listening on IPC socket in 0.5.x

### ISSUE THREAD

**Issue by atifaziz** [zappajs/zappajs#123]
In [5.2.0](https://github.com/zappajs/zappajs/releases/tag/v5.2.0), I proposed (#116) support for listening on IPC sockets and then added it via PR #117. I'd like to do the same for 0.5.0. I have done that in a local fork and it's been tested in production since months and working like a charm. If I submit a PR, would you be willing to publish a 0.5.1?

## Background

The main software that I am using that relies on zappajs is still on 0.5.0 and upgrading it to 5.x to get the IPC support added in 5.2 proved to be more difficult than anticipated due to breaking changes. See audreyt/ethercalc#422 for the gory details. This time, I'd like to simply back-port the feature to 0.5.0. The only weird side effect of doing this, I imagine, is for someone to see a feature appear in 0.5.1 only to disappear again in all subsequent versions until 5.2.0.

**Comment by shimaore** [comment]
@atifaziz I was supposed to help with the migration of ethercalc as well but I've been swamped lately. :/

I've no issue with backporting the IPC support to 0.x, feel free to open a PR.

**Comment by atifaziz** [comment]
@shimaore Yeah, I figured you got swamped. Happens, which is why I thought this may help meanwhile for something that's supposed to be a tiny change but enables a whole scenario.

> I've no issue with backporting the IPC support to 0.x, feel free to open a PR.

Awesome! Do you mind then opening a `0.5` branch on [`v0.5.0`](https://github.com/zappajs/zappajs/releases/tag/v0.5.0) tag so I can submit the PR over that?

**Comment by shimaore** [comment]
@atifaziz I noticed there's already a [0.x branch](https://github.com/zappajs/zappajs/tree/0.x) ready for this, if you don't mind using it ;)

**Comment by atifaziz** [comment]
@shimaore You're right. Should have done my homework to check that [`0.x` is pretty much `v0.5.0`](https://github.com/zappajs/zappajs/compare/v0.5.0...0.x).
